doc/api/net.md

```diff
@@ -668,6 +671,7 @@ callback.
 added: v0.1.90
 -->
 
+* `exception` {object}
```

Comment:
Should be `{object}` in all places, only primitive values are lowercased. See [tools/doc/type-parser.js](https://github.com/nodejs/node/blob/master/tools/doc/type-parser.js) if in doubt. (The build failure in CI is due to [this check](https://github.com/nodejs/node/blob/1849a2b2e98c093e27d781bd5040f76a22e8f977/tools/doc/type-parser.js#L155-L157))